### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Tanish-Panwar** [community/community#27998] [Discussions]
I want to know What skills do I need for applying to be a Github Campus Expert,
And How I can prepare for It.

**Comment by radar-blip** [comment]
Here is some info on becoming a Campus Expert: https://github.com/shree998/Campus-Expert

**Comment by truongletann** [comment]
To become a Campus Expert, you must complete the following:

Module 0 - Getting Started
Module 1 - Introduction.
At least 3 other modules.
Please bear in mind the following requirements:

Campus Experts must be 18 years of age and over.
You must not be within 3 months of finishing your course of study.
There is a limit to the number of Campus Experts per university. This limit is dependent on several factors. If in doubt, please check through one of the support channels listed below.
It's important to remember that Campus Experts is about improving the community on your campus. The modules that you complete before completing your application should be ones that will help you meet your impact proposal: choosing a module on a topic that you are already confident in will not teach you further useful skills that you can use to help your peers. During the application to become a Campus Expert, your module choices will be evaluated against how well they address the needs of your impact proposal.

Once you have completed the necessary modules, and if you are within the requirements, you can complete the application module. Once complete, we will review your application as soon as possible.

Source: https://github.com/shree998/Campus-Expert

**Reply by Tanish-Panwar** [reply]
Thank you so much for this valuable information.
